plugin.video.documentaryheaven/addon.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.documentaryheaven" name="DocumentaryHeaven" version="1.0.1+matrix.1" provider-name="Ikosse">
+  <requires>
+    <import addon="xbmc.python"                     version="3.0.0"/>
+    <import addon="script.module.requests"    		version="2.22.0+matrix.1"/>
+    <import addon="script.module.beautifulsoup4"    version="4.3.2+matrix.1"/>
+	<import addon="plugin.video.youtube"			version="6.8.10+matrix.1"/>
+	<import addon="plugin.video.vimeo"				version="6.0.0"/>
+	<import addon="plugin.video.dailymotion_com"	version="2.4.2"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="addon.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <platform>all</platform>
+    <language></language>
```

Comment:
please drop if empty

Reply:
That's a typo, it is supposed to be English. I will fix it the next time I update add-on, if that's fine. Thank you for the review!